cocos/platform/CCGLView.h

```diff
@@ -357,6 +357,25 @@ class CC_DLL GLView : public Ref
      */
     virtual void handleTouchesCancel(int num, intptr_t ids[], float xs[], float ys[]);
 
+    /** Set window icon (implemented for windows and linux).
```

Comment:
I think it's better to add Macro define like you did in `CCGLViewImpl-desktop.h`,

same situation added macro in `CCGLView.h`
```
#if (CC_TARGET_PLATFORM == CC_PLATFORM_IOS)
    virtual void* getEAGLView() const { return nullptr; }
#endif /* (CC_TARGET_PLATFORM == CC_PLATFORM_IOS) */
```

Reply:
It will require to check platform every time when user use this function.

I'm not shure about it.

Reply:
by using macro, this function is included or not, decided on compile-process. So `check platform every time when user use this function` in run-time isn't right.

Reply:
Oh！I misunderstand this behavior, you are right, and consider it more than me.